src/other/names.jl

```diff
@@ -0,0 +1,17 @@
+# common utilities for getting and comparing column names
```

Comment:
Is it really worth a separate file? Maybe utils.jl is OK?

Reply:
It is not OK. It must be placed after all types are defined. The last type introduced is `GroupKey` in `groupeddataframe/groupeddataframe.jl`, but I did not want to put it there as it is also used in `DataFrameRow` operations.

Reply:
OK - so what do we with this? I think we can leave it as is for now.